### PR TITLE
[1.3] update NV pins

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

Update the various NVIDIA package versions for Open-CE 1.3 (adds CUDA 11.2)

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Comment by npanpaliya** [comment]
We also want to update env files to include cudatoolkit feedstock if cuda_versions=11.2. And not to build tensorrt if it is cuda 11.2 on ppc64le. @jayfurmanek - Please let me know if you want me to push this change to your PR.

**Comment by npanpaliya** [comment]
@jayfurmanek - I have pushed changes to disable TRT on ppc in your branch. Also, changed the condition for building protobuf so that we build our own protobuf 3.9 for python 3.8 as well as 3.9 (though it's not targeted for now).

**Comment by cdeepali** [comment]
This PR requires - https://github.com/open-ce/cudnn-feedstock/pull/14/

**Comment by jayfurmanek** [comment]
https://github.com/open-ce/open-ce/issues/422

File: envs/tensorrt-env.yaml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 packages:
 {% set py_ver = python | float %}
 {% if build_type == 'cuda' %}
+  - feedstock : cudatoolkit      #[cudatoolkit == "11.2"]
   - feedstock: libc_wrappers     #[x86_64 and (cudatoolkit == '11.0' or cudatoolkit == '11.2' or (py_ver < 3.8 and cudatoolkit == "10.2" ))]
 {% if py_ver < 3.8 %}
   - feedstock : cudnn
-  - feedstock : tensorrt
+    # Disable TensorRT temporarily.
```
Comment (npanpaliya) [inline envs/tensorrt-env.yaml:8]: @jayfurmanek - we need to enable this if new versions of TRT are ready.

Comment (jayfurmanek) [reply envs/tensorrt-env.yaml:8]: yes we do.
The TRT update is: https://github.com/open-ce/tensorrt-feedstock/pull/24

**Comment by npanpaliya** [comment]
Will documentation update be handled in this PR itself?

File: envs/dali-env.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ imported_envs:
 
 packages:
 {% if build_type == 'cuda' %}
+  - feedstock : cudatoolkit    #[cudatoolkit == "11.2"]
```
Comment (npanpaliya) [inline envs/dali-env.yaml:8]: Should it not come from imported tensorflow-env.yaml?

Comment (jayfurmanek) [reply envs/dali-env.yaml:8]: yes, this is not strickly needed here, but it doesn't hurt.

**Review by bnemanich — APPROVED** [review]

**Comment by jayfurmanek** [comment]
We'll handle documentation in a separate PR - this one is long enough :)

**Pull request merged**